lib/plugin.js

```diff
@@ -1,11 +1,15 @@
 import Vue from 'vue'
 import VueAnalytics from 'vue-analytics'
 
-export default (ctx, inject) => {
+export default async (ctx, inject) => {
   const options = <%= serialize(options) %>
 
+  if (typeof options.id === 'function') {
```

Comment:
This `id` option overrides vue-analytics `id` option. I suggest at least rename it to something like `asyncId`

Reply:
What's the problem with overrides?

Reply:
We are changing library consideration by passing different argument (ctx) at any point vue analytics itself may change signature.

Also async at plugin level is not a good perf idea to advice. Analytics should happen with low priority than real page content.

Reply:
Ok, changed